### ISSUE TITLE
`ERROR near "and": syntax error` while using graceful-master-takeover

### ISSUE THREAD

**Issue by choadrocker** [openark/orchestrator#929]
graceful-master-takeover succeeds but gives the following error:

```ERROR near "and": syntax error```

This is with orchestrator_3.1.0_amd64.deb (which is actually the 3.1.1 release) running on Ubuntu 16.04.5 LTS Xenial 

The command run to generate the attached stack trace was: 

```orchestrator -c graceful-master-takeover -i mysql305mgd1 -debug -stack```

Stack trace:

```
2019-07-11 17:18:31 INFO starting orchestrator, version: 3.1.0, git commit: 7faf253ffacdf0ac5642545901d4a1ce2a4b4b5c
2019-07-11 17:18:31 INFO Read config: /etc/orchestrator.conf.json
2019-07-11 17:18:31 DEBUG Hostname unresolved yet: mysql305mgd1
2019-07-11 17:18:31 DEBUG Cache hostname resolve mysql305mgd1 as mysql305mgd1
2019-07-11 17:18:31 DEBUG Connected to orchestrator backend: sqlite on /var/lib/orchestrator/orchestrator.db
2019-07-11 17:18:31 DEBUG Initializing orchestrator
2019-07-11 17:18:31 INFO Connecting to backend :3306: maxConnections: 128, maxIdleConns: 32
2019-07-11 17:18:31 INFO GracefulMasterTakeover: designated master deduced to be mysql305mgd1:3306
2019-07-11 17:18:31 DEBUG Hostname unresolved yet: 192.3.1.101
2019-07-11 17:18:31 DEBUG Cache hostname resolve 192.3.1.101 as 192.3.1.101
2019-07-11 17:18:31 INFO auditType:register-candidate instance:mysql402mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:31 INFO GracefulMasterTakeover: Will demote mysql402mgd1:3306 and promote mysql305mgd1:3306 instead
2019-07-11 17:18:31 DEBUG Hostname unresolved yet: mysql425mgd1
2019-07-11 17:18:31 DEBUG Cache hostname resolve mysql425mgd1 as mysql425mgd1
2019-07-11 17:18:31 DEBUG Hostname unresolved yet: mysql201agd1
2019-07-11 17:18:31 DEBUG Cache hostname resolve mysql201agd1 as mysql201agd1
2019-07-11 17:18:31 DEBUG Hostname unresolved yet: mysql213mgd1
2019-07-11 17:18:31 DEBUG Cache hostname resolve mysql213mgd1 as mysql213mgd1
2019-07-11 17:18:31 INFO topology_recovery: Running 1 PreGracefulTakeoverProcesses hooks
2019-07-11 17:18:31 INFO topology_recovery: Running PreGracefulTakeoverProcesses hook 1 of 1: /usr/local/bin/orchestrator_logger -m 'Planned failover proceeding on mt5_multidev'
2019-07-11 17:18:31 INFO CommandRun(/usr/local/bin/orchestrator_logger -m 'Planned failover proceeding on mt5_multidev',[])
2019-07-11 17:18:31 INFO CommandRun/running: bash /tmp/orchestrator-process-cmd-006548667
2019-07-11 17:18:31 INFO CommandRun:

2019-07-11 17:18:31 INFO CommandRun successful. exit status 0
2019-07-11 17:18:31 INFO topology_recovery: Completed PreGracefulTakeoverProcesses hook 1 of 1 in 279.929999ms
2019-07-11 17:18:31 INFO topology_recovery: done running PreGracefulTakeoverProcesses hooks
2019-07-11 17:18:31 DEBUG Hostname unresolved yet: mysql402mgd1
2019-07-11 17:18:31 DEBUG Cache hostname resolve mysql402mgd1 as mysql402mgd1
2019-07-11 17:18:31 DEBUG Hostname unresolved yet: mysql402mgd1
2019-07-11 17:18:31 DEBUG Cache hostname resolve mysql402mgd1 as mysql402mgd1
2019-07-11 17:18:31 DEBUG Hostname unresolved yet: 192.13.16.63
2019-07-11 17:18:31 DEBUG Cache hostname resolve 192.13.16.63 as 192.13.16.63
2019-07-11 17:18:31 DEBUG Hostname unresolved yet: 192.36.144.25
2019-07-11 17:18:31 DEBUG Cache hostname resolve 192.36.144.25 as 192.36.144.25
2019-07-11 17:18:31 DEBUG Hostname unresolved yet: 192.3.16.69
2019-07-11 17:18:31 DEBUG Cache hostname resolve 192.3.16.69 as 192.3.16.69
2019-07-11 17:18:31 INFO auditType:register-candidate instance:mysql305mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:31 INFO auditType:register-candidate instance:mysql305mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:31 INFO Stopped replication on mysql305mgd1:3306, Self:mysql-bin.000079:189804849, Exec:mysql-bin.000090:212386433
2019-07-11 17:18:31 INFO GracefulMasterTakeover: Will set mysql402mgd1:3306 as read_only
2019-07-11 17:18:31 INFO auditType:register-candidate instance:mysql402mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:31 INFO auditType:register-candidate instance:mysql402mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:31 INFO instance mysql402mgd1:3306 read_only: true
2019-07-11 17:18:31 INFO auditType:read-only instance:mysql402mgd1:3306 cluster:mysql402mgd1:3306 message:set as true
2019-07-11 17:18:31 INFO GracefulMasterTakeover: Will advance mysql305mgd1:3306 to master coordinates mysql-bin.000090:212386433
2019-07-11 17:18:31 INFO auditType:register-candidate instance:mysql305mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:31 INFO Will start replication on mysql305mgd1:3306 until coordinates: mysql-bin.000090:212386433
2019-07-11 17:18:31 INFO auditType:register-candidate instance:mysql305mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:31 INFO auditType:register-candidate instance:mysql305mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:31 INFO auditType:register-candidate instance:mysql305mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:31 INFO Stopped replication on mysql305mgd1:3306, Self:mysql-bin.000079:189804849, Exec:mysql-bin.000090:212386433
2019-07-11 17:18:31 INFO executeCheckAndRecoverFunction: proceeding with DeadMaster detection on mysql402mgd1:3306; isActionable?: true; skipProcesses: false
2019-07-11 17:18:31 INFO topology_recovery: detected DeadMaster failure on mysql402mgd1:3306
2019-07-11 17:18:31 INFO topology_recovery: Running 1 OnFailureDetectionProcesses hooks
2019-07-11 17:18:31 INFO topology_recovery: Running OnFailureDetectionProcesses hook 1 of 1: /usr/local/bin/orchestrator_logger -s -m 'Detected DeadMaster on mt5_multidev. Affected replicas: 1'
2019-07-11 17:18:31 INFO CommandRun(/usr/local/bin/orchestrator_logger -s -m 'Detected DeadMaster on mt5_multidev. Affected replicas: 1',[])
2019-07-11 17:18:31 INFO CommandRun/running: bash /tmp/orchestrator-process-cmd-015379678
2019-07-11 17:18:32 INFO CommandRun:

2019-07-11 17:18:32 INFO CommandRun successful. exit status 0
2019-07-11 17:18:32 INFO topology_recovery: Completed OnFailureDetectionProcesses hook 1 of 1 in 487.281931ms
2019-07-11 17:18:32 INFO topology_recovery: done running OnFailureDetectionProcesses hooks
2019-07-11 17:18:32 INFO executeCheckAndRecoverFunction: proceeding with DeadMaster recovery on mysql402mgd1:3306; isRecoverable?: true; skipProcesses: false
2019-07-11 17:18:32 INFO topology_recovery: will handle DeadMaster event on mysql402mgd1:3306
2019-07-11 17:18:32 INFO auditType:recover-dead-master instance:mysql402mgd1:3306 cluster:mysql402mgd1:3306 message:problem found; will recover
2019-07-11 17:18:32 INFO topology_recovery: Running 1 PreFailoverProcesses hooks
2019-07-11 17:18:32 INFO topology_recovery: Running PreFailoverProcesses hook 1 of 1: /usr/local/bin/orchestrator_logger -m 'Will recover from DeadMaster on mt5_multidev'
2019-07-11 17:18:32 INFO CommandRun(/usr/local/bin/orchestrator_logger -m 'Will recover from DeadMaster on mt5_multidev',[])
2019-07-11 17:18:32 INFO CommandRun/running: bash /tmp/orchestrator-process-cmd-241376933
2019-07-11 17:18:32 INFO CommandRun:

2019-07-11 17:18:32 INFO CommandRun successful. exit status 0
2019-07-11 17:18:32 INFO topology_recovery: Completed PreFailoverProcesses hook 1 of 1 in 265.927129ms
2019-07-11 17:18:32 INFO topology_recovery: done running PreFailoverProcesses hooks
2019-07-11 17:18:32 INFO topology_recovery: RecoverDeadMaster: will recover mysql402mgd1:3306
2019-07-11 17:18:32 INFO topology_recovery: RecoverDeadMaster: masterRecoveryType=MasterRecoveryGTID
2019-07-11 17:18:32 INFO topology_recovery: RecoverDeadMaster: regrouping replicas via GTID
2019-07-11 17:18:32 DEBUG Stopping 1 replicas via StopReplicationNicely
2019-07-11 17:18:32 INFO auditType:register-candidate instance:mysql305mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:32 INFO auditType:register-candidate instance:mysql305mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:32 INFO auditType:register-candidate instance:mysql305mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:32 INFO Stopped slave nicely on mysql305mgd1:3306, Self:mysql-bin.000079:189804849, Exec:mysql-bin.000090:212386433
2019-07-11 17:18:32 INFO auditType:register-candidate instance:mysql305mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:32 INFO auditType:register-candidate instance:mysql305mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:32 INFO Stopped replication on mysql305mgd1:3306, Self:mysql-bin.000079:189804849, Exec:mysql-bin.000090:212386433
2019-07-11 17:18:32 DEBUG - sorted replica: mysql305mgd1:3306 mysql-bin.000090:212386433
2019-07-11 17:18:32 DEBUG GetCandidateReplica: candidate: mysql305mgd1:3306, ahead: 0, equal: 0, late: 0, break: 0
2019-07-11 17:18:32 INFO topology_recovery: RecoverDeadMaster: promotedReplicaIsIdeal(mysql305mgd1:3306)
2019-07-11 17:18:32 DEBUG RegroupReplicasGTID: working on 0 replicas
2019-07-11 17:18:32 INFO auditType:register-candidate instance:mysql305mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:32 INFO Started replication on mysql305mgd1:3306
2019-07-11 17:18:32 INFO auditType:register-candidate instance:mysql305mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:32 DEBUG RegroupReplicasGTID: done
2019-07-11 17:18:32 INFO auditType:regroup-replicas-gtid instance:mysql402mgd1:3306 cluster:mysql402mgd1:3306 message:regrouped replicas of mysql402mgd1:3306 via GTID; promoted mysql305mgd1:3306
2019-07-11 17:18:32 INFO auditType:begin-downtime instance:mysql402mgd1:3306 cluster:mysql402mgd1:3306 message:owner: root, reason: lost-in-recovery
2019-07-11 17:18:32 INFO topology_recovery: RecoverDeadMaster: 1 postponed functions
2019-07-11 17:18:32 INFO topology_recovery: checking if should replace promoted replica with a better candidate
2019-07-11 17:18:32 INFO topology_recovery: + sanity check: found our very own server to promote; doing nothing
2019-07-11 17:18:32 INFO topology_recovery: replace-promoted-replica-with-candidate: promoted instance mysql305mgd1:3306 requires no further action
2019-07-11 17:18:32 INFO topology_recovery: promoted replica: mysql305mgd1:3306
2019-07-11 17:18:32 INFO auditType:recover-dead-master instance:mysql402mgd1:3306 cluster:mysql402mgd1:3306 message:promoted replica: mysql305mgd1:3306
2019-07-11 17:18:32 INFO topology_recovery: RecoverDeadMaster: successfully promoted mysql305mgd1:3306
2019-07-11 17:18:32 INFO topology_recovery: - RecoverDeadMaster: promoted server coordinates: mysql-bin.000079:189804849
2019-07-11 17:18:32 INFO topology_recovery: - RecoverDeadMaster: will apply MySQL changes to promoted master
2019-07-11 17:18:32 INFO auditType:register-candidate instance:mysql305mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:32 INFO Will reset replica on mysql305mgd1:3306
2019-07-11 17:18:32 INFO auditType:begin-maintenance instance:mysql305mgd1:3306 cluster:mysql402mgd1:3306 message:maintenanceToken: 10, owner: root, reason: reset replica
2019-07-11 17:18:32 INFO auditType:register-candidate instance:mysql305mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:32 INFO auditType:register-candidate instance:mysql305mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:32 INFO Stopped replication on mysql305mgd1:3306, Self:mysql-bin.000079:189804849, Exec:mysql-bin.000090:212386433
2019-07-11 17:18:32 INFO auditType:register-candidate instance:mysql305mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:32 INFO Reset slave mysql305mgd1:3306
2019-07-11 17:18:32 INFO auditType:register-candidate instance:mysql305mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:32 INFO auditType:register-candidate instance:mysql305mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:32 INFO auditType:reset-slave instance:mysql305mgd1:3306 cluster:mysql402mgd1:3306 message:mysql305mgd1:3306 replication reset
2019-07-11 17:18:32 INFO auditType:end-maintenance instance:mysql305mgd1:3306 cluster:mysql402mgd1:3306 message:maintenanceToken: 10
2019-07-11 17:18:32 INFO topology_recovery: - RecoverDeadMaster: applying RESET SLAVE ALL on promoted master: success=true
2019-07-11 17:18:33 INFO auditType:register-candidate instance:mysql305mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:33 INFO auditType:register-candidate instance:mysql305mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:33 INFO instance mysql305mgd1:3306 read_only: false
2019-07-11 17:18:33 INFO auditType:read-only instance:mysql305mgd1:3306 cluster:mysql402mgd1:3306 message:set as false
2019-07-11 17:18:33 INFO topology_recovery: - RecoverDeadMaster: applying read-only=0 on promoted master: success=true
2019-07-11 17:18:33 INFO topology_recovery: Writing KV [mysql/master/mt5_multidev:mysql305mgd1:3306 mysql/master/mt5_multidev/hostname:mysql305mgd1 mysql/master/mt5_multidev/port:3306 mysql/master/mt5_multidev/ipv4:10.3.1.101 mysql/master/mt5_multidev/ipv6:]
2019-07-11 17:18:33 INFO auditType:register-candidate instance:mysql402mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:33 INFO auditType:register-candidate instance:mysql402mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:18:33 INFO instance mysql402mgd1:3306 read_only: true
2019-07-11 17:18:33 INFO auditType:read-only instance:mysql402mgd1:3306 cluster:mysql402mgd1:3306 message:set as true
2019-07-11 17:18:33 INFO topology_recovery: - RecoverDeadMaster: applying read-only=1 on demoted master: success=true
2019-07-11 17:19:14 INFO topology_recovery: Distributing KV [mysql/master/mt5_multidev:mysql305mgd1:3306 mysql/master/mt5_multidev/hostname:mysql305mgd1 mysql/master/mt5_multidev/port:3306 mysql/master/mt5_multidev/ipv4:10.3.1.101 mysql/master/mt5_multidev/ipv6:]
2019-07-11 17:19:14 INFO topology_recovery: - RecoverDeadMaster: updating cluster_alias: mysql402mgd1:3306 -> mysql305mgd1:3306
2019-07-11 17:19:14 ERROR near "and": syntax error
goroutine 1 [running]:
runtime/debug.Stack(0x32, 0x1300, 0xc0001b9960)
	/usr/local/go/src/runtime/debug/stack.go:24 +0x9d
runtime/debug.PrintStack()
	/usr/local/go/src/runtime/debug/stack.go:16 +0x22
github.com/github/orchestrator/vendor/github.com/openark/golib/log.logErrorEntry(0x2, 0xcf0d20, 0xc00018a400, 0xc000409620, 0x5f)
	/tmp/go/src/github.com/github/orchestrator/vendor/github.com/openark/golib/log/log.go:178 +0xe7
github.com/github/orchestrator/vendor/github.com/openark/golib/log.Errore(...)
	/tmp/go/src/github.com/github/orchestrator/vendor/github.com/openark/golib/log/log.go:224
github.com/github/orchestrator/vendor/github.com/openark/golib/sqlutils.ExecNoPrepare(0xc0000ba3c0, 0xc000409620, 0x5f, 0xc00038cfd8, 0x2, 0x2, 0x0, 0x0, 0x0, 0x0)
	/tmp/go/src/github.com/github/orchestrator/vendor/github.com/openark/golib/sqlutils/sqlutils.go:331 +0x162
github.com/github/orchestrator/go/db.ExecOrchestrator(0xc000409620, 0x5f, 0xc00038cfd8, 0x2, 0x2, 0xc000167180, 0x7f4596dfa008, 0x0, 0xc00009a758)
	/tmp/go/src/github.com/github/orchestrator/go/db/db.go:342 +0xd3
github.com/github/orchestrator/go/inst.ReplaceClusterName.func1(0xbf41f9c400000008, 0xc26f30)
	/tmp/go/src/github.com/github/orchestrator/go/inst/instance_dao.go:1731 +0xf3
github.com/github/orchestrator/go/inst.ExecDBWriteFunc(0xc000417980, 0x0, 0x0)
	/tmp/go/src/github.com/github/orchestrator/go/inst/instance_dao.go:137 +0x105
github.com/github/orchestrator/go/inst.ReplaceClusterName(0xc0001b98c0, 0x13, 0xc0001b98e0, 0x13, 0x0, 0xc0004089c0)
	/tmp/go/src/github.com/github/orchestrator/go/inst/instance_dao.go:1746 +0x149
github.com/github/orchestrator/go/logic.checkAndRecoverDeadMaster.func3(0xc000502000, 0xc000508e00, 0xc000508000, 0x0, 0x0)
	/tmp/go/src/github.com/github/orchestrator/go/logic/topology_recovery.go:902 +0x2ba
github.com/github/orchestrator/go/logic.checkAndRecoverDeadMaster(0xc0000258a0, 0xe, 0xcea, 0x0, 0x0, 0x0, 0xc00039c0c0, 0x13, 0xc00027f4e0, 0xc, ...)
	/tmp/go/src/github.com/github/orchestrator/go/logic/topology_recovery.go:909 +0x77a
github.com/github/orchestrator/go/logic.executeCheckAndRecoverFunction(0xc0000258a0, 0xe, 0xcea, 0x0, 0x0, 0x0, 0xc00039c0c0, 0x13, 0xc00027f4e0, 0xc, ...)
	/tmp/go/src/github.com/github/orchestrator/go/logic/topology_recovery.go:1594 +0x43e
github.com/github/orchestrator/go/logic.ForceExecuteRecovery(...)
	/tmp/go/src/github.com/github/orchestrator/go/logic/topology_recovery.go:1696
github.com/github/orchestrator/go/logic.GracefulMasterTakeover(0xc000026700, 0x13, 0x0, 0xc000026700, 0x13, 0xb1, 0xbdefae)
	/tmp/go/src/github.com/github/orchestrator/go/logic/topology_recovery.go:1887 +0x12d9
github.com/github/orchestrator/go/app.Cli(0x7fff80dc0820, 0x18, 0x0, 0x7fff80dc083c, 0xe, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/tmp/go/src/github.com/github/orchestrator/go/app/cli.go:1502 +0x65b4
github.com/github/orchestrator/go/app.CliWrapper(0x7fff80dc0820, 0x18, 0x0, 0x7fff80dc083c, 0xe, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/tmp/go/src/github.com/github/orchestrator/go/app/cli.go:151 +0x20f
main.main()
	/tmp/go/src/github.com/github/orchestrator/go/cmd/orchestrator/main.go:159 +0xf29
2019-07-11 17:19:14 ERROR near "and": syntax error
goroutine 1 [running]:
runtime/debug.Stack(0x32, 0x1300, 0xc0001b9a00)
	/usr/local/go/src/runtime/debug/stack.go:24 +0x9d
runtime/debug.PrintStack()
	/usr/local/go/src/runtime/debug/stack.go:16 +0x22
github.com/github/orchestrator/vendor/github.com/openark/golib/log.logErrorEntry(0x2, 0xcf0d20, 0xc00018a400, 0x2, 0x2)
	/tmp/go/src/github.com/github/orchestrator/vendor/github.com/openark/golib/log/log.go:178 +0xe7
github.com/github/orchestrator/vendor/github.com/openark/golib/log.Errore(...)
	/tmp/go/src/github.com/github/orchestrator/vendor/github.com/openark/golib/log/log.go:224
github.com/github/orchestrator/go/inst.ReplaceClusterName.func1(0xbf41f9c400000008, 0xc26f30)
	/tmp/go/src/github.com/github/orchestrator/go/inst/instance_dao.go:1741 +0x11a
github.com/github/orchestrator/go/inst.ExecDBWriteFunc(0xc000417980, 0x0, 0x0)
	/tmp/go/src/github.com/github/orchestrator/go/inst/instance_dao.go:137 +0x105
github.com/github/orchestrator/go/inst.ReplaceClusterName(0xc0001b98c0, 0x13, 0xc0001b98e0, 0x13, 0x0, 0xc0004089c0)
	/tmp/go/src/github.com/github/orchestrator/go/inst/instance_dao.go:1746 +0x149
github.com/github/orchestrator/go/logic.checkAndRecoverDeadMaster.func3(0xc000502000, 0xc000508e00, 0xc000508000, 0x0, 0x0)
	/tmp/go/src/github.com/github/orchestrator/go/logic/topology_recovery.go:902 +0x2ba
github.com/github/orchestrator/go/logic.checkAndRecoverDeadMaster(0xc0000258a0, 0xe, 0xcea, 0x0, 0x0, 0x0, 0xc00039c0c0, 0x13, 0xc00027f4e0, 0xc, ...)
	/tmp/go/src/github.com/github/orchestrator/go/logic/topology_recovery.go:909 +0x77a
github.com/github/orchestrator/go/logic.executeCheckAndRecoverFunction(0xc0000258a0, 0xe, 0xcea, 0x0, 0x0, 0x0, 0xc00039c0c0, 0x13, 0xc00027f4e0, 0xc, ...)
	/tmp/go/src/github.com/github/orchestrator/go/logic/topology_recovery.go:1594 +0x43e
github.com/github/orchestrator/go/logic.ForceExecuteRecovery(...)
	/tmp/go/src/github.com/github/orchestrator/go/logic/topology_recovery.go:1696
github.com/github/orchestrator/go/logic.GracefulMasterTakeover(0xc000026700, 0x13, 0x0, 0xc000026700, 0x13, 0xb1, 0xbdefae)
	/tmp/go/src/github.com/github/orchestrator/go/logic/topology_recovery.go:1887 +0x12d9
github.com/github/orchestrator/go/app.Cli(0x7fff80dc0820, 0x18, 0x0, 0x7fff80dc083c, 0xe, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/tmp/go/src/github.com/github/orchestrator/go/app/cli.go:1502 +0x65b4
github.com/github/orchestrator/go/app.CliWrapper(0x7fff80dc0820, 0x18, 0x0, 0x7fff80dc083c, 0xe, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/tmp/go/src/github.com/github/orchestrator/go/app/cli.go:151 +0x20f
main.main()
	/tmp/go/src/github.com/github/orchestrator/go/cmd/orchestrator/main.go:159 +0xf29
2019-07-11 17:19:14 INFO topology_recovery: Running 1 PostMasterFailoverProcesses hooks
2019-07-11 17:19:14 INFO topology_recovery: Running PostMasterFailoverProcesses hook 1 of 1: /usr/local/bin/orchestrator_logger -s -p -c graceful-master-takeover -m 'Recovered from DeadMaster on mt5_multidev. Failed: mysql402mgd1:3306; Promoted: mysql305mgd1:3306'
2019-07-11 17:19:14 INFO CommandRun(/usr/local/bin/orchestrator_logger -s -p -c graceful-master-takeover -m 'Recovered from DeadMaster on mt5_multidev. Failed: mysql402mgd1:3306; Promoted: mysql305mgd1:3306',[])
2019-07-11 17:19:14 INFO CommandRun/running: bash /tmp/orchestrator-process-cmd-316668352
2019-07-11 17:19:15 INFO CommandRun:

2019-07-11 17:19:15 INFO CommandRun successful. exit status 0
2019-07-11 17:19:15 INFO topology_recovery: Completed PostMasterFailoverProcesses hook 1 of 1 in 567.361145ms
2019-07-11 17:19:15 INFO topology_recovery: done running PostMasterFailoverProcesses hooks
2019-07-11 17:19:15 INFO Topology recovery: {"Id":5,"UID":"1562865512335784484:2ac411a302c9749265fe32bdcfa72298e2bf10d083e9c6d3555c98c0f5461f92","AnalysisEntry":{"AnalyzedInstanceKey":{"Hostname":"mysql402mgd1","Port":3306},"AnalyzedInstanceMasterKey":{"Hostname":"","Port":0},"ClusterDetails":{"ClusterName":"mysql402mgd1:3306","ClusterAlias":"mt5_multidev","ClusterDomain":"","CountInstances":5,"HeuristicLag":0,"HasAutomatedMasterRecovery":true,"HasAutomatedIntermediateMasterRecovery":true},"AnalyzedInstanceDataCenter":"n9","AnalyzedInstanceRegion":"","AnalyzedInstancePhysicalEnvironment":"m","IsMaster":true,"IsCoMaster":false,"LastCheckValid":true,"LastCheckPartialSuccess":true,"CountReplicas":1,"CountValidReplicas":1,"CountValidReplicatingReplicas":1,"CountReplicasFailingToConnectToMaster":0,"CountDowntimedReplicas":0,"ReplicationDepth":0,"SlaveHosts":[{"Hostname":"mysql305mgd1","Port":3306}],"IsFailingToConnectToMaster":false,"Analysis":"DeadMaster","Description":"","StructureAnalysis":null,"IsDowntimed":false,"IsReplicasDowntimed":false,"DowntimeEndTimestamp":"","DowntimeRemainingSeconds":0,"IsBinlogServer":false,"PseudoGTIDImmediateTopology":false,"OracleGTIDImmediateTopology":true,"MariaDBGTIDImmediateTopology":false,"BinlogServerImmediateTopology":false,"CountLoggingReplicas":1,"CountStatementBasedLoggingReplicas":0,"CountMixedBasedLoggingReplicas":0,"CountRowBasedLoggingReplicas":1,"CountDistinctMajorVersionsLoggingReplicas":1,"CountDelayedReplicas":0,"CountLaggingReplicas":0,"IsActionableRecovery":true,"ProcessingNodeHostname":"n9orch303mgd1","ProcessingNodeToken":"bcb4efce1aa2b703124d8e234dc03d3fb9856b663ba1ffdc1d41a0c9d45f2fac","CountAdditionalAgreeingNodes":0,"StartActivePeriod":"","SkippableDueToDowntime":false,"GTIDMode":"ON","MinReplicaGTIDMode":"ON","MaxReplicaGTIDMode":"ON","MaxReplicaGTIDErrant":"","CommandHint":"graceful-master-takeover","IsReadOnly":false},"SuccessorKey":{"Hostname":"mysql305mgd1","Port":3306},"SuccessorAlias":"","IsActive":false,"IsSuccessful":true,"LostReplicas":[],"ParticipatingInstanceKeys":[],"AllErrors":[],"RecoveryStartTimestamp":"","RecoveryEndTimestamp":"","ProcessingNodeHostname":"","ProcessingNodeToken":"","Acknowledged":false,"AcknowledgedAt":"","AcknowledgedBy":"","AcknowledgedComment":"","LastDetectionId":0,"RelatedRecoveryId":0,"Type":"MasterRecovery","RecoveryType":"MasterRecoveryGTID"}
2019-07-11 17:19:15 INFO topology_recovery: Running 2 PostFailoverProcesses hooks
2019-07-11 17:19:15 INFO topology_recovery: Running PostFailoverProcesses hook 1 of 2: /usr/local/bin/orchestrator_logger -m 'Post failover run orc_post_failover'
2019-07-11 17:19:15 INFO CommandRun(/usr/local/bin/orchestrator_logger -m 'Post failover run orc_post_failover',[])
2019-07-11 17:19:15 INFO CommandRun/running: bash /tmp/orchestrator-process-cmd-080648735
2019-07-11 17:19:15 INFO CommandRun:

2019-07-11 17:19:15 INFO CommandRun successful. exit status 0
2019-07-11 17:19:15 INFO topology_recovery: Completed PostFailoverProcesses hook 1 of 2 in 233.25387ms
2019-07-11 17:19:15 INFO topology_recovery: Running PostFailoverProcesses hook 2 of 2: /usr/local/bin/orc_post_failover -F mysql402mgd1 -S mysql305mgd1
2019-07-11 17:19:15 INFO CommandRun(/usr/local/bin/orc_post_failover -F mysql402mgd1 -S mysql305mgd1,[])
2019-07-11 17:19:15 INFO CommandRun/running: bash /tmp/orchestrator-process-cmd-186226930
2019-07-11 17:19:15 INFO CommandRun:

2019-07-11 17:19:15 INFO CommandRun successful. exit status 0
2019-07-11 17:19:15 INFO topology_recovery: Completed PostFailoverProcesses hook 2 of 2 in 112.44285ms
2019-07-11 17:19:15 INFO topology_recovery: done running PostFailoverProcesses hooks
2019-07-11 17:19:15 INFO topology_recovery: Waiting for 1 postponed functions
2019-07-11 17:19:15 DEBUG PostponedFunctionsContainer: waiting on 1 postponed functions
2019-07-11 17:19:15 DEBUG PostponedFunctionsContainer: done waiting
2019-07-11 17:19:15 INFO topology_recovery: Executed 1 postponed functions
2019-07-11 17:19:15 INFO topology_recovery: Executed postponed functions: regroup-replicas-gtid mysql305mgd1:3306
2019-07-11 17:19:15 INFO auditType:register-candidate instance:mysql402mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:19:15 DEBUG ChangeMasterTo: will attempt changing master on mysql402mgd1:3306 to mysql305mgd1:3306, mysql-bin.000079:189804849
2019-07-11 17:19:15 INFO ChangeMasterTo: Changed master on mysql402mgd1:3306 to: mysql305mgd1:3306, mysql-bin.000079:189804849. GTID: true
2019-07-11 17:19:15 INFO auditType:register-candidate instance:mysql402mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:19:15 INFO auditType:register-candidate instance:mysql402mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:19:15 DEBUG ChangeMasterTo: will attempt changing master credentials on mysql402mgd1:3306
2019-07-11 17:19:15 INFO ChangeMasterTo: Changed master credentials on mysql402mgd1:3306
2019-07-11 17:19:15 INFO auditType:register-candidate instance:mysql402mgd1:3306 cluster:mysql402mgd1:3306 message:neutral
2019-07-11 17:19:15 INFO topology_recovery: Running 1 PostGracefulTakeoverProcesses hooks
2019-07-11 17:19:15 INFO topology_recovery: Running PostGracefulTakeoverProcesses hook 1 of 1: /usr/local/bin/orchestrator_logger -s -m 'Graceful takeover'
2019-07-11 17:19:15 INFO CommandRun(/usr/local/bin/orchestrator_logger -s -m 'Graceful takeover',[])
2019-07-11 17:19:15 INFO CommandRun/running: bash /tmp/orchestrator-process-cmd-554408873
2019-07-11 17:19:16 INFO CommandRun:

2019-07-11 17:19:16 INFO CommandRun successful. exit status 0
2019-07-11 17:19:16 INFO topology_recovery: Completed PostGracefulTakeoverProcesses hook 1 of 1 in 671.355216ms
2019-07-11 17:19:16 INFO topology_recovery: done running PostGracefulTakeoverProcesses hooks
mysql305mgd1:3306
mysql-bin.000079:189804849
2019-07-11 17:19:16 DEBUG Promoted mysql305mgd1:3306 as new master. Binlog coordinates at time of promotion: mysql-bin.000079:189804849
```

**Comment by shlomi-noach** [comment]
Thank you! Addressed by https://github.com/github/orchestrator/pull/931
